src/requirements/data/colleges/ag.ts

```diff
@@ -107,10 +100,447 @@ const calsIntroductoryLifeSciencesOrBiologyRequirement: CollegeOrMajorRequiremen
   description:
-    '18 credits in at least three disciplines of which six credits must be introductory life sciences/biology and three credits in chemistry or physics and a quantitative literacy course.',
+    '18 credits in at least three disciplines of which six credits must be introductory life sciences/biology and three credits in chemistry or physics',
   source:
     'https://cals.cornell.edu/undergraduate-students/student-services/degree-requirements/graduation-requirements/distribution-requirements',
-  fulfilledBy: 'self-check',
+  checker: includesWithSingleRequirement(
+    'AEM 2100',
+    'AEM 3100',
+    'AEM 3100',
+    'ALS 3601',
+    'ANSC 1100',
+    'ANSC 2120',
+    'ANSC 2150',
+    'ANSC 2210',
+    'ANSC 2400',
+    'ANSC 2410',
+    'ANSC 2650',
+    'ANSC 3200',
+    'ANSC 3300',
+    'ANSC 3410',
+    'ANSC 3700',
+    'ANSC 3920',
+    'ANSC 3921',
+    'ANSC 4400',
+    'ASTRO 1101',
+    'ASTRO 1103',
+    'ASTRO 1104',
+    'ASTRO 2201',
+    'ASTRO 2202',
+    'ASTRO 2211',
+    'ASTRO 2212',
+    'ASTRO 4410',
+    'ASTRO 4431',
+    'ASTRO 4445',
+    'BEE 4590',
+    'BIOAP 1100',
+    'BIOAP 3110',
+    'BIOAP 3160',
+    'BIOAP 3190',
+    'BIOAP 3410',
+    'BIOAP 4580',
+    'BIOEE 1540',
+    'BIOEE 2070',
+    'BIOEE 2740',
+    'BIOEE 3510',
+    'BIOEE 3690',
+    'BIOEE 4460',
+    'BIOEE 4461',
+    'BIOEE 4550',
+    'BIOEE 4690',
+    'BIOEE 4790',
+    'BIOEE 4920',
+    'BIOEE 4930',
+    'BIOG 1102',
+    'BIOG 1104',
+    'BIOG 1106',
+    'BIOG 1190',
+    'BIOG 1191',
+    'BIOG 2011',
+    'BIOG 2020',
+    'BIOG 3000',
+    'BIOG 3020',
+    'BIOMG 1150',
+    'BIOMG 1320',
+    'BIOMG 2800',
+    'BIOMG 2801',
+    'BIOMG 3300',
+    'BIOMG 3340',
+    'BIOMG 3350',
+    'BIOMG 3800',
+    'BIOMG 4320',
+    'BIOMG 4330',
+    'BIOMG 4350',
+    'BIOMG 4380',
+    'BIOMG 4400',
+    'BIOMG 4450',
+    'BIOMG 4610',
+    'BIOMG 4860',
+    'BIOMI 2790',
+    'BIOMI 2900',
+    'BIOMI 2911',
+    'BIOMI 2920',
+    'BIOMI 3310',
+    'BIOMI 3910',
+    'BIOMI 3940',
+    'BIOMI 4090',
+    'BIOMI 4310',
+    'BIOMI 4480',
+    'BIOMI 4850',
+    'BIOMI 6901',
+    'BIOMS 4400',
+    'BIONB 2210',
+    'BIONB 2220',
+    'BIONB 3215',
+    'BIONB 3280',
+    'BIONB 3690',
+    'BIONB 4200',
+    'BIONB 4370',
+    'BIONB 4910',
+    'BIONB 4970',
+    'BIOPL 2210',
+    'BIOPL 2410',
+    'BIOPL 2420',
+    'BIOPL 2421',
+    'BIOPL 2484',
+    'BIOPL 2490',
+    'BIOPL 3420',
+    'BIOPL 3421',
+    'BIOPL 3800',
+    'BIOPL 4220',
+    'BIOPL 4480',
+    'BIOPL 4620',
+    'BIOPL 4826',
+    'BIOPL 4831',
+    'BSOC 2101',
+    'BSOC 4691',
+    'BTRY 3010',
+    'BTRY 3020',
+    'BTRY 3100',
+    'BTRY 4090',
+    'BTRY 4090',
+    'BTRY 4100',
+    'BTRY 4830',
+    'BTRY 4840',
+    'BTRY 4940',
+    'BTRY 6840',
+    'CEE 3040',
+    'CHEM 1150',
+    'CHEM 1560',
+    'CHEM 1570',
+    'CHEM 2070',
+    'CHEM 2080',
+    'CHEM 2090',
+    'CHEM 2150',
+    'CHEM 2870',
```

Comment:
I think you're missing CHEM 2510
![Screen Shot 2021-03-23 at 10 23 27 AM](https://user-images.githubusercontent.com/54298311/112161748-cf6e6680-8bc1-11eb-98c9-f73f54b16d37.png)